.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,49 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔
+
+---
+
+<!-- Thanks for helping grpc-swift! Before you submit your issue, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->
+
+### New Issue Checklist
+
+- [ ] Updated grpc-swift to the latest version
```

Comment:
I would add a line confirming that they checked that the versions of their runtime library (dependency) and their protoc plugin match.